groups/config.yaml

```diff
@@ -0,0 +1,10 @@
+# Configuration for the kubernetes.io Google Groups setup
+
+# file path of the google groups service account token
+token-file: k8s-infra-test-project-1896690daeb3.json
```

Comment:
The fact this says "test" suggests we should move it

Reply:
We can rename this when we have the new service account @thockin